### DISCUSSION TITLE
如何添加阅读过但是没有在正文中引用过的参考文献？

### DISCUSSION THREAD

**Discussion by Sergey3557** [tuna/thuthesis#872] [Q&A] [answered]
我知道thuthesis的模版可以利用bibtex插入参考文献，但是必须在正文中引用过才能够显示在参考文献中，直接写在ref.bib文件中的参考文献是无法显示在参考文献中的。

请问有没有大佬知道怎么修改一下代码让阅读过但是没有在正文中直接引用的参考文献也显示出来呢？

**Comment by note286** [comment]
如果没有引用的文献不应该出现在参考文献列表中。

**Reply by Sergey3557** [reply]
但是我看我们本科生的论文写作指南要求里写着可以加入一些直接阅读过的文献呀？

**Reply by Harry-Chen** [reply, marked as answer]
如果一定需要，可以使用 `\nocite{id}`

**Reply by Sergey3557** [reply]
<img width="552" alt="截屏2023-04-29 10 56 13" src="https://user-images.githubusercontent.com/132117030/235280415-28a6a8c2-7e39-4bf4-9a93-90c4f5cb3744.png">

**Reply by Sergey3557** [reply]
> 如果一定需要，可以使用 `\nocite{id}`

请问加在哪个文件的哪个位置呢？

**Reply by Harry-Chen** [reply]
这里的要求是并列的，意思是阅读过 && 引用过

**Reply by note286** [reply]
如果你不引用，意味着这个文献和你的论文没什么关系，也就是说你没有参考，为什么放在参考文献列表呢？

**Reply by Sergey3557** [reply]
> 这里的要求是并列的，意思是阅读过 && 引用过

好的，我明白了，感谢🙏

**Reply by Harry-Chen** [reply]
> > 如果一定需要，可以使用 `\nocite{id}`
> 
> 请问加在哪个文件的哪个位置呢？

在正文的任意位置。我想这随便尝试一下就知道了吧。

**Reply by Sergey3557** [reply]
> 如果你不引用，意味着这个文献和你的论文没什么关系，也就是说你没有参考，为什么放在参考文献列表呢？

确实😂，感谢🙏
